internal/cli/kraft/run/runner_kraftfile_unikraft.go

```diff
@@ -147,6 +149,12 @@ func (runner *runnerKraftfileUnikraft) Prepare(ctx context.Context, opts *RunOpt
 		"CONFIG_LIBVFSCORE_AUTOMOUNT_CI_EINITRD",
 	)
 
+	noRandom := t.KConfig().AllNoOrUnset("CONFIG_LIBUKRANDOM_LCPU") && cpuid.CPU.Has(cpuid.RDRAND) ||
```

Comment:
This one looks like different logic altoghether 🤔

Reply:
did some logic toke reduce the number of negations, will simplfy in another way